src/config.h

```diff
@@ -101,6 +101,27 @@
 #endif
 #endif
 
+/* Test for __builtin_prefetch() */
+/* Supported in LLVM since 2.9:
+ * https://releases.llvm.org/2.9/docs/ReleaseNotes.html */
+/* Supported in GCC since 3.1:
+ * https://gcc.gnu.org/gcc-3.1/changes.html */
```

Comment:
```suggestion
/* Test for __builtin_prefetch()
 * Supported in LLVM since 2.9: https://releases.llvm.org/2.9/docs/ReleaseNotes.html
 * Supported in GCC since 3.1: https://gcc.gnu.org/gcc-3.1/changes.html. */
```
let us use redis comments style.

And gcc 3.1 is too old, 3.2 release notes said `Fixed prefetch code generation`, maybe we can use higher version

Reply:
@ShooterIT should we move to 3.2 or higher?

Reply:
Actually redis requires gcc version at least 4.1, AFAIK, because https://github.com/redis/redis/commit/445a4b669a3a7232a18bf23340c5f7d580aa92c7 remove mutex fallback.
I don't know which version is better, and don't know what 3.2 fixed, maybe we can leave it as it is or update it to 3.2, cc @sundb

Reply:
@fcostaoliveira @ShooterIT I think 4.9 is enough, which is the lowest version we support.

Reply:
addressed in the latest commit